new-pipeline/style/dashboards.css

```diff
@@ -43,6 +43,13 @@
     z-index: 1;
     top: 5px;
 }
+
+/** Max Width Setting For e10s **/
+.btn-group ul
+{
+    max-width: 40vw !important;
```

Comment:
Can you target just e10s' ul? You should be able to use adjacent sibling selectors to find the `ul` of the sibling of `#filter-e10s`

Reply:
I have added the e10s specific css